run_dir/design/worksets.html

```diff
@@ -36,6 +30,6 @@ <h1 id="{{ worksets }}" class="worksets_page_heading"><span id="page_title">All
 
 </div> <!-- #page_content -->
 
-<script src="/static/js/list.min.js"></script>
+<script src="/static/js/jquery.dataTables.min.js?v={{ gs_globals['git_commit'] }}"></script>
```

Comment:
I don't think we need the `?v={{ gs_globals['git_commit'] }}` for libraries that we're including, do we? This code shouldn't be changing much.

Mind you, I guess it doesn't really hurt either.